container_files/public_html/doc/builtin/sql/ValueExpression.md

```diff
@@ -695,3 +703,16 @@ log to the console to aid debugging. Documentation for this object can be found
 
 You can also take a look at the ![](%%nblink _tutorials/Executing JavaScript Code Directly in SQL Queries Using the jseval Function Tutorial) for examples of how to use the `jseval` function.
 
+### Non-simplified arguments
+
+If the first character of the argument string is `!`, then non-simplified
+arguments are used.  These are harder to work with in Javascript, but allow
+for the entire set of values in MLDB to be represented, especially structured
+values or those with repeated columns or multiple timestamps per value.
+
+For example, the following query yields the same as `(SELECT x:1, y:2)`,
+in other words it doesn't mess around with the values:
+
+```sql
+SELECT jseval('return row;', '!row', {*}) AS * FROM (SELECT x:1, y:2)"
```

Comment:
This example seems artificial.  If it is, I would consider adding an example that is more realistic (e.g. the use case that forces us to add the non-simplified argument).